setup.py

```diff
@@ -38,12 +39,11 @@
         'Development Status :: 2 - Pre-Alpha',
         'Intended Audience :: Developers',
         'Natural Language :: English',
-        "Programming Language :: Python :: 2",
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
```

Comment:
I guess we want to support only python 3.7. In this case we should remove these lines.